packages/harpstrata/src/harpstrata/HarpStrata/getHarpStrata/index.ts

```diff
@@ -18,7 +18,7 @@ export const getHarpStrata = (props: HarpStrataProps): HarpStrata => {
 
   const degreeMatrix = getDegreeMatrix(halfstepIndexMatrix, rootOffset)
   const pitchMatrix = getPitchMatrix(halfstepIndexMatrix, pitchId)
-  const isActiveComplex = getIsActiveComplex({
+  const { activeDegreeIds, activePitchIds } = getIsActiveComplex({
```

Comment:
`getIsActiveComplex` needs to be renamed, or even removed given the function which sources it doesn't even have it's return value manipulated.

Reply:
It was removed